### PR TITLE
BW - 1304 get /runs endpoint

### PR DESCRIPTION
**Pull request by kpierre13**

### CODE REVIEW

File: service/src/main/java/bio/terra/cbas/controllers/RunsApiController.java [hunk 1]
```diff
@@ -22,6 +27,25 @@ public RunsApiController(CromwellServerConfiguration cromwellConfig) {
...
 
+  @Override
+  public ResponseEntity<RunLogResponse> getRun() {
+    List<LogRunRequest> runs = new ArrayList<>();
+    String runId = UUID.randomUUID().toString();
+    String name = "CBAS";
+    Date now = new Date();
+
+    runs.add(
+        (new LogRunRequest()
```
Comment (fredlas) [inline service/src/main/java/bio/terra/cbas/controllers/RunsApiController.java:38]: You can skip this paren pair (unless you are keeping consistent style with how other HTTP request handlers are written in this codebase, or something like that)

File: service/src/main/java/bio/terra/cbas/controllers/RunsApiController.java [hunk 2]
```diff
@@ -22,6 +27,25 @@ public RunsApiController(CromwellServerConfiguration cromwellConfig) {
...
+            .runId(runId)
+            .state(RunState.UNKNOWN)
+            .workflowUrl("urlHere")
```
Comment (fredlas) [inline service/src/main/java/bio/terra/cbas/controllers/RunsApiController.java:41]: (Just to confirm) is this an intentional placeholder for something that can't be done yet? (Same question for "params" below)

**Review by fredlas — APPROVED** [review]
Looks good; all I had were a couple of style suggestions and a question/confirmation.

**Review by salonishah11 — APPROVED** [review]

File: service/src/main/java/bio/terra/cbas/controllers/RunsApiController.java (hunk 2 above)
Comment (kpierre13) [reply service/src/main/java/bio/terra/cbas/controllers/RunsApiController.java:41]: Yes, definitely an intentional placeholder. The next ticket will be the actual wiring to Cromwell.

File: service/src/main/java/bio/terra/cbas/controllers/RunsApiController.java [hunk 3]
```diff
@@ -22,6 +27,25 @@ public RunsApiController(CromwellServerConfiguration cromwellConfig) {
     this.cromwellConfig = cromwellConfig;
   }
 
+  @Override
+  public ResponseEntity<RunLogResponse> getRun() {
+    List<LogRunRequest> runs = new ArrayList<>();
+    String runId = UUID.randomUUID().toString();
```
[the remark this answers is not in the document]
Comment (kpierre13) [reply service/src/main/java/bio/terra/cbas/controllers/RunsApiController.java:33]: Do you mind explaining what you mean by, 'directly where they're used'? Do you mean in-line with where they appear later on in the code?

[the remark this answers is not in the document]
Comment (kpierre13) [reply service/src/main/java/bio/terra/cbas/controllers/RunsApiController.java:33]: Maybe something like ```runs.add(
        new LogRunRequest()
            .runId(UUID.randomUUID().toString())```?

**Pull request merged**